fpsim/locations/location_template.py

```diff
@@ -0,0 +1,826 @@
+        'end_year'               : 2020, # End year of simulation
+        'timestep'               : 1, # The simulation timestep in months
+        'method_timestep'        : 1, # How many simulation timesteps to go for every method update step
+        'seed'                   : 1, # Random seed
+        'verbose'                : 1, # How much detail to print during the simulation
+        'track_switching'        : 0, # Whether to track method switching
+        'track_as'               : 0, # Whether to track age-specific channels
+
+        # Age limits (in years)
+        'method_age'             : 15,
+        'age_limit_fecundity'    : 50,
+        'max_age'                : 99,
+
+        # Durations (in months)
+        'switch_frequency'       : 12, # How frequently to check for changes to contraception
+        'end_first_tri'          : 3,
+        'preg_dur_low'           : 9,
+        'preg_dur_high'          : 9,
+        'postpartum_dur'         : 35,
+ ### context-specific       'breastfeeding_dur_mu'   : 19.66828, # Location parameter of gumbel distribution. Requires children's recode DHS file, see data_processing/breastfeedin_stats.R 
+ ### context-specific       'breastfeeding_dur_beta' : 7.2585, # Scale parameter of gumbel distribution. Requires children's recode DHS file, see data_processing/breastfeedin_stats.R 
+        'max_lam_dur'            : 5, # Duration of lactational amenorrhea
+
+        # Pregnancy outcomes
+ ### context-specific       'abortion_prob'             : 0.08, # From https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4712915/
+### context-specific        'twins_prob'                : 0.015, # From https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0025239
+        'LAM_efficacy'              : 0.98, # From Cochrane review: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6823189/
+        'maternal_mortality_factor' : 1,
+
+         # Fecundity and exposure
+        'fecundity_var_low'      : 0.7,
+        'fecundity_var_high'     : 1.1,
+        'high_parity'            : 4,
+        'high_parity_nonuse'     : 0.6,
+        'primary_infertility'    : 0.05,
+        'exposure_factor'        : 1.0, # Overall exposure correction factor
+
+        # MCPR
+### context-specific        'mcpr_growth_rate'       : 0.02, # The year-on-year change in MCPR after the end of the data
+### context-specific        'mcpr_max'               : 0.90, # Do not allow MCPR to increase beyond this
+### context-specific        'mcpr_norm_year'         : 2018, # Year to normalize MCPR trend to 1
+    }
+    return scalar_pars
+
+
+def data2interp(data, ages, normalize=False):
+    ''' Convert unevenly spaced data into an even spline interpolation '''
+    model = si.interp1d(data[0], data[1])
+    interp = model(ages)
+    if normalize:
+        interp = np.minimum(1, np.maximum(0, interp))
+    return interp
+
+
+def filenames():
+    ''' Data files for use with calibration, etc -- not needed for running a sim '''
```

Comment:
The code to make these is all in data processing folder, right? I would say that here.